src/assessment.js

```diff
@@ -27,7 +27,7 @@ var q1 = /* your answer here */;
 //
 //     `['alderaan', 'deathStar', 'fire', 'global']`
 //
-var q2 = [/* your answer here */];
+var q2 = [deathStar, fire, global];
```

Comment:
The `global` scope will not have access to `laserQuantity` because it is hidden inside the `deathStar` function.